src/ensembl/src/content/app/custom-download/containers/content/customDownloadContentHelper.ts

```diff
@@ -0,0 +1,36 @@
+import JSONValue from 'src/shared/types/JSON';
+
+export const flattenObject = (
+  objectOrArray: JSONValue,
+  prefix = '',
+  formatter = (k: string) => k
+) => {
+  if (!objectOrArray) {
+    return {};
+  }
+  const nestedFormatter = (k: string) => '.' + k;
+
+  const nestElement = (prev: any, value: any, key: any): JSONValue =>
```

Comment:
any, any, any => JSONValue, JSONValue, string?

Reply:
TypeScript is not happy with that. This is just a temporary solution until the backend is available.